python/tvm/relay/op/dyn/image/_image.py

```diff
@@ -67,10 +57,19 @@ def resize_shape_func(attrs, inputs, _):
     Shape function for dyn.image.resize op.
     """
     layout = attrs.layout
-    if layout == 'NHWC':
-        out = [_NHWC_resize_shape_func(inputs[0].shape, inputs[1], convert(len(inputs[0].shape)))]
-    elif (layout == 'NCHW') or nchw_pack_layout(layout) or nchw_xc_layout(layout):
-        out = [_NCHW_resize_shape_func(inputs[0].shape, inputs[1], convert(len(inputs[0].shape)))]
+    if nchw_pack_layout(layout) or nchw_xc_layout(layout):
+        out = [_resize_shape_func(inputs[0].shape, inputs[1], convert(len(inputs[0].shape)),
+                                  convert(2), convert(3), convert(1))]
```

Comment:
Does the shape function work for `NCHWc` layout?

Reply:
It should. The shape function copies the entire shape by iterating over ndim, so all dimensions are copied before any swapping happens.

Reply:
I see. 
btw. why do we need `out[channel_axis] = int64(dshape[channel_axis])` since it's already copied?

Reply:
I guess it is redundant, I will remove it